helm_deploy/prisoner-content-hub-proxy/templates/post-deploy-job.yaml

```diff
@@ -14,6 +14,5 @@ spec:
       restartPolicy: Never
       containers:
       - name: post-deploy-job
-        image: "byrnedo/alpine-curl"
-        # Has an entrypoint of curl ENTRYPOINT ["/usr/bin/curl"]
-        command: ["{{ .Values.ingress.hostName }}/{{ .Values.npr.path }}/npr-stream"]
+        image: "curlimages/curl"
+        command: ["-l", "{{ .Values.ingress.hostName }}/{{ .Values.npr.path }}/npr-stream"]
```

Comment:
:heart: for the `HEAD` request. Should we use `--head` in preference to `-l` for clarity? I'm a big fan of using long-options in scripts to improve readability.

Reply:
Makes sense and done :)